c2rust-transpile/tests/snapshots/snapshots__transpile-linux@macros.c.snap

```diff
@@ -31,11 +31,13 @@ pub unsafe extern "C" fn size_of_const() -> std::ffi::c_int {
     return SIZE as std::ffi::c_int;
 }
 pub const SIZE: usize = unsafe { ::core::mem::size_of::<[std::ffi::c_int; 10]>() };
+pub const POS: [std::ffi::c_char; 3] =
+    unsafe { *::core::mem::transmute::<&[u8; 3], &[std::ffi::c_char; 3]>(b"\"]\0") };
```

Comment:
As a follow-up, we might be able to clean these up by switching from byte literals to C string literals. I found an [old issue](https://github.com/immunant/c2rust/issues/22) on this.

Reply:
Yup, definitely something I'd like to do.